### PR TITLE
Admin & Staff adjustments

### PR DESCRIPTION
**Pull request by skull132**
Features:
- [x] Removal of BOREALIS (python module) as it's not used.
- [x] Removes ToR ban feature in lieu of IPIntel.
- [x] New BOREALIS config to alert staff if server starts as hidden.
- [x] Adminhelps now inform admins on discord if dibsed (when they were sent to discord anyways).
- [x] Adds hub visibility to the server access control panel.
- [x] Adds mirror ban spotting via ban panel. It now redirects to the linked ban if one is found.
- [x] CCIAA now get alerted as to how many of them are online and active when receiving faxes and emergency messages via Discord.
- [x] Removed unused C/C++ libraries. The socket_talk module is a generic UDP shipper, of which Arrow implemented a better version. lib nudge is not even compiled for use. lib_nudge module is uncompiled and no longer used, as we use cURL for the bot.
- [x] Removed depracted APIs and config settings related to the previous point.
- [x] Whitelisted jobs now appear properly in the job selection window as `[WHITELISTED]`.
- [x] Job ban reasons can now be viewed from player preferences window.
- [x] ~~Await admin approval for final CCIAA requests and implement.~~ RIP CCIAA.
- [x] Fix age bans for jobs and antags (dynamic ones, ofc).
- [x] Implement https://forums.aurorastation.org/viewtopic.php?f=18&t=8283

Important internal changes:
* The jobban API has been merged with the ageban API. It was stupid that we had 2 different ageban APIs that were separated from the jobban API. It's not the prettiest solution, but it works relatively effectively IMO.
* I nuked a whole bunch of unnecessary config and bot related files. All of them legacy, none of them necessary for our purposes.
* Most of this is tested as functional.

### CODE REVIEW

**Comment by skull132** [comment]
Marked as feedback required due to the nuking of multiple old libs/APIs.

**Comment by skull132** [comment]
Admins denied changes requested by FT. PR is complete and awaiting review.

**Comment by skull132** [comment]
hokay. Ready for review, I promise I won't touch it anymore. ~~What is atomic PR-ing.~~

File: html/changelogs/skull132-adminstuff.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: Skull132
+delete-after: True
+
+changes:
+  - rscadd: "You can now view the reason for your active job ban by clicking on the [BANNED] text on job/role selection."
+  - tweak: "Whitelisted jobs now display as [WHITELISTED] instead of [BANNED] if you area short a whitelist."
```
Comment (Lohikar) [inline html/changelogs/skull132-adminstuff.yml]: Typo.

**Pull request merged**